cmd/minikube/cmd/config/disable.go

```diff
@@ -33,6 +33,9 @@ var addonsDisableCmd = &cobra.Command{
 		}
 
 		addon := args[0]
+		if addon == "heapster" {
+			exit.WithCodeT(exit.Unavailable, "There is no heapster addon")
```

Comment:
I would say since we automaticly select metric-server for them instead of heapster, we ask them to disable metric
how about this.
```
exit.WithCodeT(exit.Unavailable, "The heapster addon is deprecicated will try to disable metrics-server instead  ")
```

Reply:
Thanks!

I changed to the following.
```
exit.WithCodeT(exit.Unavailable, "The heapster addon is depreciated. please try to disable metrics-server instead")
```

If any idea, please let me know.